pageserver/src/tenant.rs

```diff
@@ -3405,30 +3410,42 @@ fn run_initdb(
         initdb_bin_path, initdb_target_dir, initdb_lib_dir,
     );
 
-    let initdb_output = Command::new(&initdb_bin_path)
+    let _permit = INIT_DB_SEMAPHORE.acquire().await;
+
+    let initdb_command = tokio::process::Command::new(&initdb_bin_path)
```

Comment:
Interesting we don't close fds for this like we do for walredo; I was expecting that. 

Why shouldn't we close?